src/types/onyx/Policy.ts

```diff
@@ -1212,6 +1212,9 @@ type Connections = {
 
     /** Sage Intacct integration connection */
     intacct: Connection<SageIntacctConnectionData, SageIntacctConnectionsConfig>;
+
+    /** Advanced Approvals integration connection */
+    approvals: Connection<undefined, undefined>;
```

Comment:
This shouldn't be added here as approvals is not an external connection 🤔

Reply:
Does it error when it's removed?

Reply:
yes, the types don't match in `PolicyAccountingPage.tsx` in more than one place, what results in errors

Reply:
Actually I found another (less complicated) solution; since AdvancedApprovals is an upgradable, but internal function it's natural to put its key in another place: `INTERNAL` instead of `CONNECTIONS` key in `CONST.tsx`. What do you think?